ml4h/arguments.py

```diff
@@ -376,6 +384,16 @@ def parse_args():
         default='3M',
     )
 
+    # Arguments for explorations/infer_medians
+    parser.add_argument('--analyze_ground_truth', default=True, help='Whether or not to filter by images with ground truth segmentations, for comparison')
+    parser.add_argument('--dates_file', help='File containing dates for each sample_id')
```

Comment:
Can we re-use the more general existing arg app_csv for this? Just trying to mitigate argument explosion.